bigtable/client/table.h

```diff
@@ -117,6 +117,8 @@ class Table {
    *     successfully apply the mutation given the current policies. The
    *     exception contains a copy of the original mutation, in case the
    *     application wants to retry, log, or otherwise handle the failure.
+   *
```

Comment:
Other comments include `**Example**` heading, should this have it too?

Reply:
Done.